packages/sandbox/package.json

```diff
@@ -28,14 +28,15 @@
     "@aws-sdk/types": "^3.378.0",
     "@aws-sdk/client-cloudformation": "^3.421.0",
     "@parcel/watcher": "^2.3.0",
-    "glob": "^10.2.7",
     "debounce-promise": "^3.1.2",
-    "parse-gitignore": "^2.0.0",
-    "open": "^9.1.0"
+    "glob": "^10.2.7",
+    "open": "^9.1.0",
+    "parse-gitignore": "^2.0.0"
   },
   "devDependencies": {
     "@types/debounce-promise": "^3.1.6",
-    "@types/parse-gitignore": "^1.0.0"
+    "@types/parse-gitignore": "^1.0.0",
+    "@types/uuid": "^9.0.7"
```

Comment:
we can use crypto.randomUUID to avoid pulling in this dep

Reply:
this is actually needed in platform-core (moved there) and since it's core, we should use the browser compatible libraries and I'm not sure if node's crypto is.